app/src/main/kotlin/at/bitfire/davdroid/db/WebDavDocumentDao.kt

```diff
@@ -21,8 +23,15 @@ interface WebDavDocumentDao {
     @Query("SELECT * FROM webdav_document WHERE mountId=:mountId AND (parentId=:parentId OR (parentId IS NULL AND :parentId IS NULL)) AND name=:name")
     fun getByParentAndName(mountId: Long, parentId: Long?, name: String): WebDavDocument?
 
-    @Query("SELECT * FROM webdav_document WHERE parentId=:parentId")
-    fun getChildren(parentId: Long): List<WebDavDocument>
+    @RawQuery
+    fun getChildren(query: RoomRawQuery): List<WebDavDocument>
+
+    fun getChildren(parentId: Long, orderBy: String = "name ASC"): List<WebDavDocument> {
```

Comment:
If the caller is responsible to pass only validated SQL clauses, this must be documented in KDoc so that the responsibility is clear.